Configurable s3 storage class (#4300)
grafana/mimir: CHANGELOG.md

# Changelog

## main / unreleased

> * [FEATURE] Add capability to configure the used S3 Storage Class [#3438] (https://github.com/grafana/mimir/pull/4300) (@inbarpatashnik)

Reviewer: Could you please move this underneath the 'Grafana Mimir' heading?

> * [FEATURE] Add capability to configure the used S3 Storage Class [#3438] (https://github.com/grafana/mimir/pull/4300) (@inbarpatashnik)

Reviewer: Please also follow the same format as for other changelog entries:

```suggestion
* [FEATURE] Add capability to configure the S3 storage class. #3438
```

### Grafana Mimir

> * [FEATURE] Add capability to configure the S3 storage class. #3438

Reviewer: ```suggestion
* [FEATURE] Added `-<prefix>.s3.storage-class` flag to configure the S3 storage class for objects written to S3 buckets. #3438
```

Reviewer: Hope you don't mind @inbarpatashnik, going to commit this minor edit so we can get this merged :)

> * [FEATURE] Added `-<prefix>.s3.storage-class` flag to configure the S3 storage class for objects written to S3 buckets. #3438

Reviewer 2: Can you move this next to the other `[FEATURE]` entries.

### Grafana Mimir

> * [FEATURE] Add capability to configure the S3 storage class. #3438
> 

Reviewer: [nit] Blank line not required.

```suggestion
* [FEATURE] Add capability to configure the S3 storage class. #3438
```
